.github/workflows/kind-e2e.yaml

```diff
@@ -17,18 +17,19 @@ jobs:
     strategy:
       fail-fast: false # Keep running if one leg fails.
       matrix:
-        k8s-version:
-        - v1.20.7
-        - v1.21.1
-        - v1.22.2
+        kingress:
+        - contour
+        - kourier
+        - istio
+        - gateway-api
```

Comment:
Can we somehow continue to print the k8s version to the jobname as well?

Reply:
Hmm... No idea for now. Adding KinD test for gateway-api is not urgent so I reverted it for now. I will add the test after going through the syntax of github action :sweat_smile: